File: TUF/Numbers/CheckAutomorphicNumber.java
import java.util.Scanner;

public class CheckAutomorphicNumber {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a number to check: ");
        int num = input.nextInt();
        input.close();
        System.out.println(isAutomorphicNumber(num));
    }

    private static boolean isAutomorphicNumber(int num) {
        int finalNum = num * num;
        return findLastDigit(finalNum) == findLastDigit(num);
    }

    private static int findLastDigit(int num) {
        return num % 10;
    }
}

File: TUF/Numbers/CheckAutomorphicNumber.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
function isAutomorphicNumber(num) {
    var finalNum = num * num;
    return findLastDigit(finalNum) == findLastDigit(num);
}
function findLastDigit(num) {
    return num % 10;
}
var num = 76;
console.log("".concat(num, " is an automorphic number: ").concat(isAutomorphicNumber(num) ? "Yes" : "No"));
